bsoid_app/bsoid_analysis.py

```diff
@@ -20,46 +20,46 @@ def streamlit_run(pyfile):
 st.markdown('Step 1: Pick the directory and workspace to analyze.')
 st.markdown('Step 2: Once input, select the type of results to analyze using the sidebar modules.')
 st.text('')
-working_dir = st.text_input('Enter B-SOiD __output directory__ from using the B-SOiD --version 2.0 App')
+WORKING_DIR = st.text_input('Enter B-SOiD __output directory__ from using the B-SOiD --version 2.0 App')
```

Comment:
How is this file used? I don't know if these changes are intentional

Reply:
bsoid_analysis.py is actually downstream of the steps I was focusing on for this PR, let's revisit this if its still an issue in the next PR